CMSgov/onemac
# OY2-12767 - OneMAC status to display for corresponding SEATool status

Story: https://qmacbis.atlassian.net/browse/OY2-12767
Endpoint: https://d16ajra8g3llrp.cloudfront.net

### Details
SEA Tool sends Package Status as a number used in a lookup table to view the status text. We were hoping to receive the text, but are now going to do the number/text translation on the oneMAC side.

### Changes
- the SEA Tool status conversion table now uses the status lookup ids instead of the text
- null status from SEA Tool results in "Submitted" in oneMAC

### Implementation Notes
- It is tempting to add a migration script to this story to convert statuses already in oneMAC.  However, this data is already behind a "feature flag" and there is already a story to migrate data and this can be part of that.

### Test Plan
1. Login as someone to submit packages
2. submit a package, note the ID/Number used
3. Navigate to Package Dashboard
4. Package status should read "Package In Review" - might need to refresh.  This is the SEA Tool boomerang status
5. Test the Lambda  with this JSON (substitute in the ID_Number of your package and convert the plan type accordingly:
  CHIP_SPA: "124",
  SPA: "125",
  WAIVER: "122",
  WAIVER_APP_K: "123",
)
{
    "value": "{\"payload\":{\"ID_Number\":\"MI.3232\",\"Plan_Type\":122,\"Action_Type\":76,\"Alert_90_Days_Date\":1632355200000,\"SPW_Status_ID\":4,\"End_Date\":1761596720000}}"
}
6. Package Dashboard should now read "Package Approved" for Status type 4
